W-13743185 Exchange usage and engagement content for GA jh (#255)
mulesoft/docs-exchange: modules/ROOT/pages/metrics-for-a-specific-asset-version.adoc

= Metrics for a Specific Asset and Version
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images/

> Administrators and asset owners with contributor permission can view usage details for a specific API specification fragment and REST API asset and each version on the asset details page. This page identifies asset versions with overall high usage and versions with low usage. The usage rates can help you understand the performance and stickiness of specific asset versions within your organization and enable you to make informed decisions about reuse and improvements. 

Reviewer: ```suggestion
Administrators and asset owners with contributor permission can view usage details for a specific API specification fragment, REST API asset. and their versions on the asset details page. This page identifies asset versions with overall high usage and versions with low usage. The usage rates can help you understand the performance and stickiness of specific asset versions within your organization and enable you to make informed decisions about reuse and improvements. 
```

Author: Keeping "and" since you can view metrics for both types of assets. I will change "each version" to "their version".

> Administrators and asset owners with contributor permission can view usage details for a specific API specification fragment, REST API asset. and their versions on the asset details page. This page identifies asset versions with overall high usage and versions with low usage. The usage rates can help you understand the performance and stickiness of specific asset versions within your organization and enable you to make informed decisions about reuse and improvements. 

Reviewer: ```suggestion
Administrators and asset owners with contributor permission can view usage details for a specific API specification fragment, REST API asset, and their versions on the asset details page. This page identifies asset versions with overall high usage and versions with low usage. The usage rates can help you understand the performance and stickiness of specific asset versions within your organization and enable you to make informed decisions about reuse and improvements. 
```

The following graphic shows the asset details page with usage and engagement metrics:

image::ex-metrics-for-asset-version.png[Metrics for an asset and version]

The following data is collected and analyzed:

* Version
+
The version of the asset.
* Created by
+
The name of the person who created this version of the asset.
* Created on
+
The date that the asset was created.
* Lifecycle state
+
Which state the asset is currently in. The states are In Development, Stable, or Deprecated.
* Conformance 
+

> Conformance status appears for all REST APIs and AsyncAPIs. Status can be Conformant, Not Conformant, and Not Validated.

Reviewer: ```suggestion
Conformance status appears for all REST APIs and AsyncAPIs. The status can be Conformant, Not Conformant, and Not Validated.
```

* Engagement contribution
+
The engagement contribution is the percentage of usage of the given asset patch or minor version with regards to its respective minor or major version.
* Downloads
+
The number of times the asset was downloaded from Exchange. You can sort by ascending or descending order. 
* API Designer imports
+

> The number of times the asset was imported into Design Center. You can sort by ascending or descending order.

Reviewer: ```suggestion
The number of times the asset was imported in to Design Center. You can sort by ascending or descending order.
```

* Studio imports
+

> The number of times the asset was imported into Studio. You can sort by ascending or descending order.

Reviewer: ```suggestion
The number of times the asset was imported in to Studio. You can sort by ascending or descending order.
```

* All-time stats
** Dependents
+

> The number of dependent assets since the version was created.

Reviewer: ```suggestion
The number of dependent assets from the time the version was created.
```

** Contracts
+

> The number of approved contracts since the version was created.

Reviewer: ```suggestion
The number of approved contracts from the time the version was created.
```

== View Asset Version Engagement

To view usage and engagement metrics for the asset versions of API specification fragments and REST APIs:

. In the Exchange catalog, select an API specification fragment or REST API.
+
The asset details page displays the asset metadata.
. In the navigation bar, click *Asset versions engagement*.
+
image::ex-nav-asset-versions-engagement.png[The navigation bar with Asset Versions Engagement option,15%,5%]
+
The *Asset versions engagement* page displays.

> . If you want to filter by date, select a date range from 7 days, 30 days, or 60 days.

Reviewer 2: ```suggestion
. If you want to filter by date, select a date range from 7 days, 30 days, or 90 days.
```
